coordinate.go

```diff
@@ -101,15 +102,15 @@ func (a *Agent) runNodePing(node *api.Node) {
 	// Update the node's health based on the results of the ping.
 	if err == nil {
 		if err := a.updateHealthyNode(node, kvClient, key, kvPair); err != nil {
-			a.logger.Printf("[WARN] error updating node: %v", err)
+			a.logger.Warn("error updating node", "error", err)
 		}
 		if err := a.updateNodeCoordinate(node, rtt); err != nil {
-			a.logger.Printf("[WARN] could not update coordinate for node %q: %v", node.Node, err)
+			a.logger.Warn("could not update coordinate for node", "node", hclog.Fmt("%q: %v", node.Node, err))
 		}
 	} else {
-		a.logger.Printf("[WARN] could not ping node %q: %v", node.Node, err)
+		a.logger.Warn("could not ping node", "node", hclog.Fmt("%q: %v", node.Node, err))
```

Comment:
Here as well, can you split "node" k/v pair into something like `"node", node.Node, "error", err`?